components/brave_wallet_ui/components/extension/allow-add-change-network-panel/index.tsx

```diff
@@ -47,36 +39,24 @@ export interface Props {
 function AllowAddChangeNetworkPanel (props: Props) {
   const {
     siteOrigin,
-    selectedNetwork,
```

Comment:
Since we don't need `selectedNetwork` any more, can we remove it from the `interface` as well?
Will need to remove it from being passed into the component in `Storybook` and in the `Panel` container as well.